src/bidiTab/mapperTabPage.ts

```diff
@@ -70,6 +70,11 @@ export function log(logType: LogType, ...messages: unknown[]) {
     return;
   }
 
+  // If `sendDebugMessage` is defined, send the log message there.
+  (globalThis as unknown as Window).sendDebugMessage?.(
```

Comment:
does `as global.Window` work by any chance? To leverage the definitions from `bidiTab.ts`

Reply:
done